pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -117,6 +116,14 @@ func (jm *CronJobController) syncAll() {
 	js := jl.Items
 	glog.V(4).Infof("Found %d jobs", len(js))
 
+	sjl, err := jm.kubeClient.BatchV2alpha1().CronJobs(metav1.NamespaceAll).List(metav1.ListOptions{})
+	if err != nil {
+		glog.Errorf("Error listing cronjobs: %v", err)
```

Comment:
Can you change glog.Errorf calls to utilruntime.HandleError (the latter is ratelimited among other things)